### DISCUSSION TITLE
Welcome to box-cli-maker Discussions!

### DISCUSSION THREAD

**Discussion by Delta456** [Delta456/box-cli-maker#18] [General]
## 👋 Welcome!
 I will be using Discussions feature of GitHub for users to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Report bugs and discussion how to solve them.
  * Showcase your CLIs made with `box-cli-maker`!

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ssss536** [comment]
Your are welcome
